docs/client/full-api/api/accounts.md

```diff
@@ -255,7 +252,64 @@ Example:
       passwordSignupFields: 'USERNAME_AND_OPTIONAL_EMAIL'
     });
 
-{{> autoApiBox "Accounts.validateNewUser"}}
+<h2 id="advanced_accounts_api"><span>Advanced Accounts</span></h2>
```

Comment:
I'm not a huge fan of the heading "Advanced Accounts" - it could maybe be more descriptive like "accounts for multiple servers" or something, but I can't really come up with anything much better on the spot.